src/frequenz/sdk/actor/_background_service.py

```diff
@@ -167,6 +170,8 @@ def cancel(self, msg: str | None = None) -> None:
         Args:
             msg: The message to be passed to the tasks being cancelled.
         """
+        _logger.info("Service %s cancelled, reason: %s", self.name, msg)
```

Comment:
Maybe it would be a bit easier to the eyes like this?

```suggestion
        _logger.info("Service %s cancelled%s", self.name, msg, f": {msg}" if msg is not None else "")
```